blazor/input-mask/mask-configuration.md

```diff
@@ -112,4 +112,39 @@ The following example demonstrates the MaskedTextBox with customized prompt char
 
 
 
-![Blazor MaskedTextBox with Prompt Character](./images/blazor-maskedtextbox-with-prompt-char.png)
\ No newline at end of file
+![Blazor MaskedTextBox with Prompt Character](./images/blazor-maskedtextbox-with-prompt-char.png)
+
+## Include Literals in the Value
+
+By default, you will get the raw value as [Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Inputs.SfMaskedTextBox.html#Syncfusion_Blazor_Inputs_SfMaskedTextBox_Value) of the component. Now, you can get the component Value with mask literals by enabling [EnableLiterals](https://www.syncfusion.com/blazor-components/blazor-input-mask) property.
+
+The following example demonstrates how to obtain the value with mask literals.
+
+{% highlight Razor %}
+
+{% include_relative code-snippet/mask-configuration/EnableLiterals.razor %}
+
+{% endhighlight %}
+
+![Blazor MaskedTextBox with Included Literals](./images/EnableLiterals.gif)
+
+## Prompt Placeholder
```

Comment:
Make as sub section as per mentioned in the outline. Follow the proper casing provided in the outline document